config/settings.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 return [
-    'base'          => [
+    'pub_base'          => [
```

Comment:
"pub" and "openpub" have been just interchangeably in the past (just as "pdc" and "openpdc") But for consistency we would prefer using "openpub" and "openpdc". Could you change `pub_base` into `openpub_base`?

Reply:
Understandable! The change has been made https://github.com/OpenWebconcept/plugin-openpub-base/pull/8/commits/d93550997dc58efc2195be1228be2345a26feb45